lib/scanner/csv-scanner/csv-scanner.h

```diff
@@ -80,9 +79,7 @@ gboolean csv_scanner_scan_next(CSVScanner *pstate);
 gboolean csv_scanner_is_scan_finished(CSVScanner *pstate);
 gchar *csv_scanner_dup_current_value(CSVScanner *self);
 
-void csv_scanner_input(CSVScanner *pstate, const gchar *input);
-gboolean csv_scanner_parse_input(CSVScanner *pstate);
-void csv_scanner_state_init(CSVScanner *pstate, CSVScannerOptions *options);
+void csv_scanner_init(CSVScanner *pstate, CSVScannerOptions *options, const gchar *input);
 void csv_scanner_state_clean(CSVScanner *pstate);
 
```

Comment:
isn't state_clean the pair of csv_scanner_init()? the naming does not suggest that.